src/cadenza.js

```diff
@@ -133,6 +133,18 @@ globalThis.cadenza = Object.assign(
  * _Note:_ Since numbers in JavaScript are Double values ([more info on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number#number_encoding)),
  * for Long variables, the API is currently limited to the Double value range.
  */
+/**
+ * @typedef Feature - A [GeoJSON](https://geojson.org/) feature object
+ * @property {any[]} id - The id of the feature
+ * @property {Geometry} geometry - The geometry
+ * @property {Record<string, string>} properties - The properties
+ * @property {Number | undefined} area - The area of the polygon
+ * @property {Number | undefined} length - The length of the line
```

Comment:
I think this should be like this:
```javascript
 * @property {number} [area] - The area of a `Polygon` feature
 * @property {number} [length] - The area of a `LineString` feature
```
* Primitive types are lower case.
* Optional props are marked with [].